interface-definitions/include/pki/openvpn_tls-auth.xml.i

```diff
@@ -0,0 +1,14 @@
+<!-- include start from pki/openvpn_tls-auth.xml.i -->
```

Comment:
Well I am unsure if there is a need for an "auth key" CLI node - it is the same as an OpenVPN site2site key if I remember correctly. We should find a common name for the CLI node.